lib/catalog/martyrology.ts

```diff
@@ -4111,6 +4490,18 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
     visitation_of_mary: {
       name: 'Visitation of the Blessed Virgin Mary',
     },
+    // src:
+    // - mr_fr_1982_ed2_coutances
+    // - https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/saint-vital-abbe-de-savigny
+    // - https://en.wikipedia.org/wiki/Vitalis_of_Savigny
+    vitalis_of_savigny_bishop: {
+      canonizationLevel: CanonizationLevels.Saint,
+      name: 'Vitalis of Savigny',
+      titles: [Title.Bishop],
```

Comment:
```suggestion
      titles: [Title.Abbot],
```

Reply:
Indeed, and we have to rename the ID too.